src/app/components/LoginPage/login.scss

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css?family=Mulish');
+
+:local {
+  .popup {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 1920px;
+    height: 1080px;
+    background: rgba(0, 0, 0, 0.4);
+    z-index: 2;
+    mix-blend-mode: normal;
+  }
+
+  .login {
+    position: absolute;
+    width: 600px;
+    height: 352px;
+    left: 660px;
+    top: 96px;
+    background: #FFFFFF;
+    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
```

Comment:
Avoid `position: absolute` for simple components.

There are no good reasons to use `position: absolute` here.